widget/widgetAssets/schedulePlaybook.directive.js

```diff
@@ -91,8 +123,10 @@
       function loadScheduleData() {
         $http({
           method: 'GET',
-          url: API.WORKFLOW + 'api/scheduled/?format=json&name=Investigate_Outbreak-Alerts'
+          url: API.WORKFLOW + 'api/scheduled/?format=json&name=Investigate_' + scope.jsonData.name
         }).then(function (response) {
+          scope.params.form.schedule.kwargs.exit_if_running = true;
+          scope.params.form.scheduled = 'Y'
```

Comment:
please use boolean here

Reply:
same code used in UI code .. hence followed same terminology